src/models.py

```diff
@@ -15,11 +15,14 @@ class Chunk(BaseModel):
     id: str
     text: str
     chunk_type: BlockType
-    # TODO: do we want multiple headings here? this is what docling does.
     heading: Optional["Chunk"] = None
-    # Bounding boxes can be arbitrary polygons according to Azure
-    # TODO: is this true according to the backend or frontend?
-    bounding_boxes: Optional[list[list[tuple[float, float]]]]
+    # Bounding boxes can be arbitrary polygons according to Azure, but we assume
+    # they're quadrilaterals in the frontend PDF viewer code. So it makes sense to
+    # carry this assumption back to here.
+    # See https://github.com/climatepolicyradar/navigator-frontend/blob/b7f25a3fd2ed815fbb09416b9a046f0f1c41a0cf/src/hooks/usePDFPreview.ts#L20
+    bounding_boxes: Optional[
+        list[Annotated[list[tuple[float, float]], Field(min_length=4, max_length=4)]]
+    ]
```

Comment:
could/should we add some soft validation which logs or throws a warning when the bounding box isn't a neat quadrilateral? or is the extra computational overhead used to run that test on every chunk unnecessary?

Reply:
I thought about that, but I think this option (throwing a validation error, causing the code to break) is better as we have limited observability of our pipeline already and I wonder if any soft validation would just be ignored. Wdyt?

Reply:
yeah, sounds reasonable to me!